bin/adhoc-scripts/drainAgent.py

```diff
@@ -202,12 +202,12 @@ def getCondorJobs():
     """
     jobDict = {}
     schedd = condor.Schedd()
-    jobs = schedd.xquery('WMAgent_AgentName == "WMAgent"', ['WMAgent_RequestName', 'JobStatus'])
+    jobs = schedd.xquery('WMA_AgentName == "WMAgent"', ['WMA_RequestName', 'JobStatus'])
```

Comment:
These parameters are different from environment variables. Can you please roll this back?